addons/binding/org.openhab.binding.vera2/src/main/java/org/openhab/binding/vera2/handler/VeraBridgeHandler.java

```diff
@@ -0,0 +1,200 @@
+/**
+ * Copyright (c) 2010-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.vera2.handler;

```

Comment:
I already thought about it. Yes, it's better to rename it to "Vera".
1.x binding was called mios, but the company was recently renamed to Vera, so there will be no conflict between the names of the bindings.